src/controllers/controller.bar.js

```diff
@@ -112,10 +112,9 @@ module.exports = function(Chart) {
 			var me = this;
 			var meta = me.getMeta();
 			var yScale = me.getScaleForId(meta.yAxisID);
-			var base = yScale.getBaseValue();
-			var original = base;
+			var base = 0;
```

Comment:
Not sure if that are my local changes but it also seems that bars are now always stacked, even with `stacked: false` on the scale.